src/memefactory/server/twitter_bot.cljs

```diff
@@ -95,12 +99,12 @@
          (-> (server-utils/get-ipfs-binary-file ipfs-hash)
              (.then (fn [image-tar-file-content]
                       (when-not just-log-tweet?
-                        (.then (first-tar-obj image-tar-file-content)
-                               (fn [image-file-content]
-                                 (-> (upload-file-to-twitter twitter-obj image-file-content)
-                                     (.then (fn [media-id] (resolve media-id)))
-                                     (.catch (fn [e] (log/error "Error uploading media to twitter" {:registry-entry registry-entry
-                                                                                                    :ipfs-hash ipfs-hash})))))))))))))))
+                        (-> (first-tar-obj image-tar-file-content)
```

Comment:
use promise-> macro

Reply:
also what is the difference here now we have the default rejection handler?

Reply:
Preference - avoid boilerplate code, error checking and logging handled. YMMV